test/test_sample.py

```diff
@@ -74,6 +74,7 @@ def test_bernoulli_good(stanfile: str):
 
     for i in range(bern_fit.runset.chains):
         csv_file = bern_fit.runset.csv_files[i]
+        # NB this assumes we're not using threads for chains
```

Comment:
why is this relevant?

Reply:
The tests run in alphabetical order, and so the new Pathfinder test which compiled with `STAN_THREADS` was leaving behind a executable that this test was picking up. But when `STAN_THREADS` is enabled, all the chains output to the same output file, so this check was failing. 

I figured it was good to note for a future explorer (for example, if `STAN_THREADS` becomes the default one day)

Reply:
the comment is too cryptic - how about - "NB:  separate output files when `STAN_THREADS` is False."